config/locales/hello.en.yml

```diff
@@ -13,6 +13,9 @@ en:
         subject: "Reset Password Instructions"
 
     entities:
+      current_user:
```

Comment:
@marcelobarreto thank you for this PR :heart: 
I see you're adding a new translation, but shouldn't you be renaming the translation that already existed? :sweat_smile: